docs/concepts/dqm.rst

```diff
@@ -4,13 +4,31 @@
 Discrete Quadratic Models 
 =========================
 
-The discrete quadratic model (DQM) is a polynomial over discrete variables with
-terms all of degree two or less.  Suppose that there are :math:`N` discrete
-variables :math:`\bf{d}_i`, each with :math:`n_i` cases.  Conceptually, the
-cases may represent any collection of discrete values, such as ``{red, green,
-blue, yellow}`` or ``{3.2, 67}``.  Using a binary variable :math:`x_{i,u}` to
-indicate whether discrete variable :math:`\bf{d}_i` is set to case :math:`u`,
-the objective function can be expressed by the equation:
+The discrete quadratic model (DQM) is a polynomial over *discrete* variables, 
+with all terms of degree two or less, where a discrete variable represents some 
+collection of distinct values, such as ``{red, green, blue, yellow}`` or 
+``{3.2, 67}``, which are called the variable's *cases*.  
+
+A discrete quadratic model may be defined as
+
+.. math::
+
+    H(\bf{d}) = \sum_{i} a(\bf{d}_i) + \sum_{i,j} b(\bf{d}_i,\bf{d}_j) + c
```

Comment:
I believe that the "a" and "b" functions need subscripts:
```suggestion
    H(\bf{d}) = \sum_{i} a_i(\bf{d}_i) + \sum_{i,j} b_{i,j}(\bf{d}_i,\bf{d}_j) + c
```

Reply:
yes, typo